### PR TITLE
added 404 not found page

### PR DESCRIPTION
**Pull request by constantincrismaruvitagroup**

### CODE REVIEW

File: src/app/_common-components/not-found/not-found.component.html [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+    <div class="not-found-code">404</div>
+    <div class="message-info message-info-line message-info__first-line"> {{ 'captions.NotFound.ops' | translate }}</div>
+    <div class="message-info message-info-line"> {{ 'captions.NotFound.wentWrong' | translate }}</div>
+    <div class="message-details"> {{ 'captions.NotFound.notPandemic' | translate }}</div>
+    <div class="message-info message-info-line back-to-button"> {{ 'captions.NotFound.backTo' | translate }}</div>
+    <button mat-flat-button color="primary" (click)="navigateToDashboard()">
+        {{ 'captions.mainMenuDashboard' | translate }}
+    </button>
+    <div class="need-help"> {{ 'captions.NotFound.needHelp' | translate }}</div>
+    <a routerLink="/about">{{ 'captions.NotFound.contactUs' | translate }}</a>
```
Comment (adriandumavitagroup) [inline src/app/_common-components/not-found/not-found.component.html]: change it to this please [routerLink]="['/about']" so we have it consistent

File: src/app/_common-components/not-found/not-found.component.html [hunk 2]
```diff
@@ -1,2 +1,12 @@
-<h1>{{ 'PageNotFound' | translate }}</h1>
-<a routerLink="/">{{ 'goToDashboard' | translate }}</a>
+<div class="not-found-container">
+    <div class="not-found-code">404</div>
+    <div class="message-info message-info-line message-info__first-line"> {{ 'captions.NotFound.ops' | translate }}</div>
+    <div class="message-info message-info-line"> {{ 'captions.NotFound.wentWrong' | translate }}</div>
+    <div class="message-details"> {{ 'captions.NotFound.notPandemic' | translate }}</div>
+    <div class="message-info message-info-line back-to-button"> {{ 'captions.NotFound.backTo' | translate }}</div>
+    <button mat-flat-button color="primary" (click)="navigateToDashboard()">
```
Comment (adriandumavitagroup) [inline src/app/_common-components/not-found/not-found.component.html]: add [routerLink] for the dashboard link, just like at the about route

File: src/app/_common-components/not-found/not-found.component.ts [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
   selector: 'app-not-found',
   templateUrl: './not-found.component.html',
   styleUrls: ['./not-found.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class NotFoundComponent {}
+export class NotFoundComponent {
+  constructor(private router: Router) { }
+  
+  navigateToDashboard() {
```
Comment (adriandumavitagroup) [inline src/app/_common-components/not-found/not-found.component.ts]: you can delete this, after you add the dashboard [routerLink] in html

File: src/app/_common-components/not-found/not-found.component.ts [hunk 4]
```diff
@@ -1,9 +1,16 @@
 import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-not-found',
   templateUrl: './not-found.component.html',
   styleUrls: ['./not-found.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
```
Comment (adriandumavitagroup) [inline src/app/_common-components/not-found/not-found.component.ts]: changeDetection is not needed here

File: src/assets/i18n/en.json [hunk 5]
```diff
@@ -2481,7 +2481,13 @@
...
     "TaskExport.personPhoneOwner": "Person phone owner",
     "TaskExport.personEmailAddress": "Person email address",
-    "TaskExport.personOtherContactDetails": "Person contact details"
+    "TaskExport.personOtherContactDetails": "Person contact details",
+    "NotFound.ops": "Ooops!",
+    "NotFound.wentWrong": "Something went wrong",
+    "NotFound.notPandemic": "It is not a pandemic",
+    "NotFound.backTo": "Back to",
+    "NotFound.needHelp": "need help?",
+    "NotFound.contactUs": "Contact Us"
```
Comment (adriandumavitagroup) [inline src/assets/i18n/en.json]: remove the translations from here and put them at the bottom of the file in custom-translations.json and run the docker comands

File: src/app/_common-components/not-found/not-found.component.scss [hunk 6]
```diff
@@ -1 +1,52 @@
-/* stylelint-disable */
+@use '/src/variables.scss' as *;
+@use '@angular/material' as mat;
+
+:host {
+    display: flex;
+    min-height: 100%;
+    background-image: url('/assets/img/background-about.png');
```
Comment (adriandumavitagroup) [inline src/app/_common-components/not-found/not-found.component.scss]: make a pull from development please ; the background image has been change, the way we do it ; look at the about component

File: src/app/_common-components/not-found/not-found.component.html [hunk 7]
```diff
@@ -1,2 +1,17 @@
...
+        <div class="not-found-code">404</div>
+        <div class="message-info message-info-line message-info__first-line">
+            {{ 'NotFound.ops' | translate }}
+        </div>
+        <div class="message-info message-info-line"> {{ 'NotFound.wentWrong' | translate }}</div>
+        <div class="message-details"> {{ 'NotFound.notPandemic' | translate }}</div>
+        <div class="message-info message-info-line back-to-button"> {{ 'NotFound.backTo' | translate }}</div>
+        <button mat-flat-button color="primary" [routerLink]="['/']">
+            {{ 'captions.mainMenuDashboard' | translate }}
+        </button>
```
Comment (adriandumavitagroup) [inline src/app/_common-components/not-found/not-found.component.html]: if this tag has [routerLink], make it a tag (anchor) ... not a button

**Review by adriandumavitagroup — APPROVED** [review]

**Review by danielrus — APPROVED** [review]

**Review by sandorbanvitagroup — APPROVED** [review]

**Pull request merged**